### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by 0xRishav**

### CODE REVIEW

File: src/components/alert/Alert.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.Success {
```
Comment (poojasabnani) [inline src/components/alert/Alert.css:1]: class names can begin with small letters

File: src/components/input/Input.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.form__group {
+  position: relative;
+  padding: 15px 0 0;
```
Comment (ritikarp) [inline src/components/input/Input.css]: Use relative units like rem instead of px.

File: src/components/avatar/Avatar.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import "./Avatar.css";
```
Comment (poojasabnani) [inline src/components/avatar/Avatar.js:1]: individual css imports can be added to style.css and only style.css can be imported in app.js

File: src/components/input/Input.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+}
+.form__field {
+  font-family: inherit;
+  width: 100%;
+  border: 0;
+  border-bottom: 2px solid #9b9b9b;
+  outline: 0;
+  font-size: 1.3rem;
+  color: black;
+  padding: 7px 0;
```
Comment (ritikarp) [inline src/components/input/Input.css]: Here as well

File: src/components/badge/Badge.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  right: 5%;
+  padding: 10px;
+  border-radius: 20px;
+}
+
+.Badge__span--red {
+  background-color: #f93154;
+}
+.Badge__span--green {
+  background-color: #00b74a;
```
Comment (poojasabnani) [inline src/components/badge/Badge.css:16]: variable names can be used for colors

File: src/components/productCard/ProductCard.css [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+.Product__buttonWrapper {
+  /* display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between; */
+  margin: auto;
+}
+/* .Product__buttonWrapper > * {
+    width: 100%;
+  } */
```
Comment (ritikarp) [inline src/components/productCard/ProductCard.css]: Delete commented code

File: src/components/index.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+export { default as Avatar } from "./avatar/Avatar";
```
Comment (poojasabnani) [inline src/components/index.js:1]: really nice!

File: src/components/input/Input.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./Input.css";
+
+function Input() {
+  return (
+    <div style={{ display: "flex", flexDirection: "column" }}>
```
Comment (poojasabnani) [inline src/components/input/Input.js]: inline styling can be avoided

File: src/components/productCard/ProductCard.css [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+
+.Product__productName {
+  font-weight: 500;
+}
+.Product__price {
+  opacity: 0.9;
+}
+
+.Product__buttonWrapper {
+  /* display: flex;
```
Comment (poojasabnani) [inline src/components/productCard/ProductCard.css]: commented code can be cleaned up

File: src/components/textVideoCard/TextVideoCard.css [hunk 10]
```diff
@@ -0,0 +1,41 @@
+.textVideoCard {
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  text-align: left;
+  margin: 1rem;
+  color: #313131;
```
Comment (ritikarp) [inline src/components/textVideoCard/TextVideoCard.css]: define primary color in root to avoid repetition

**Review by ritikarp — COMMENTED** [review]
Hi @iamrishavb 
Great work 
Added some comments please check

**Review by poojasabnani — COMMENTED** [review]
Hi Rishav, nicely written code. Have suggested some naming practices and code cleanup.

**Pull request merged**